### PR TITLE
Recruit Creature Info window has an incorrect position in higher than default resolution

### PR DESCRIPTION
**Pull request by shprotru**
Yes I read CONTRIBUTING.md, but I can't resist offering an case insensitive access on *nix systems. You have an script/demo/demo_linux.sh that will dl zip, after extraction data directory named by default as "DATA" and need to rename it manually. Please accept the patch, because different HOMM2 distributives have different naming and manual renaming on *nix is a time wasting.

Fixes: #424

### CODE REVIEW

**Comment by shprotru** [comment]
how to pass:
`The command "if [[ "$TRAVIS_PULL_REQUEST" != "false" ]]; then`
or this is a magic code format checks? They are depends on script `$TRAVIS_BUILD_DIR/script/tools/clang-format-diff.py` that is not accessible in this repo. Could you help me with viewing this kind of errors?

**Comment by ihhub** [comment]
Hi @shprotru , thank you very much for your attempts! Could you please create a separate pull request for changes not related to #424? It is important to keep track of changes when they are separate.

File: fheroes2.cfg [hunk 1]
```diff
@@ -27,6 +27,7 @@ music = on
 #
 # video mode (640x480, 800x576, 1024x768, 1280x1024 or other divisible 32)
 # videomode = 640x480
+videomode = 1920x1080
```
Comment (ihhub) [inline fheroes2.cfg:30]: Could you please revert this change as we do not encourage to change configuration file settings?

Comment (shprotru) [reply fheroes2.cfg]: Sorry, I didn't notice, fixed

File: src/fheroes2/dialog/dialog_recrut.cpp [hunk 2]
```diff
@@ -458,7 +458,7 @@ void Dialog::DwellingInfo( const Monster & monster, u32 available )
     const payment_t paymentMonster = monster.GetCost();
     const Sprite & box = AGG::GetICN( ICN::RECR2BKG, 0 );
 
-    const SpriteBack back( Rect( ( display.w() - box.w() ) / 2, 16, box.w(), box.h() ) );
+    SpriteBack back( Rect( ( display.w() - box.w() ) / 2, ( display.h() - box.h() ) / 2 - ( Settings::Get().QVGA() ? 15 : 65 ), box.w(), box.h() ) );
```
Comment (ihhub) [inline src/fheroes2/dialog/dialog_recrut.cpp:461]: Could you please explain why we do not need `const` modifier here anymore?

Comment (shprotru) [reply src/fheroes2/dialog/dialog_recrut.cpp:461]: Main reason: settings could be changed in future dynamically(in new options window, for example) and we will be ready for that in this place: coords calculation would by dynamically. Secondary reason: because I don't found in project other places usage `const` modifier with SpriteBack class on static dialog.

Comment (ihhub) [reply src/fheroes2/dialog/dialog_recrut.cpp:461]: This is a local variable and we do not change this within function so keeping it `const` makes a good sense (**const**-ness principle). Please ignore other places of code. The code was written initially not by us so there are multiple development mistakes which we try to fix over time.

On the top of this, please remove QVGA() related code. We are not going to support it anymore as nowadays devices have way higher resolution than this. Our plan is that in the future we full drop QVGA support and remove all related to it source code.

File: .gitignore [hunk 3]
```diff
@@ -27,6 +27,12 @@
 *.out
 *.app
 
+#VSCode
```
Comment (ihhub) [inline .gitignore:30]: I am not sure that it is really necessary to exclude these two file types as people could use other text editors such as `nano` for example. Then we will have to add these extension as well.

Let's make this pull request simple and do not include these changes for now. You could create a separate pull request for this file if you think that this is a very important change to do :)

Comment (shprotru) [reply .gitignore]: done

File: src/fheroes2/dialog/dialog_recrut.cpp (hunk 2 above)
Comment (shprotru) [reply src/fheroes2/dialog/dialog_recrut.cpp]: done

**Comment by ihhub** [comment]
@shprotru thank you very much for the contribution to the project!
